users/users.md

```diff
@@ -210,17 +210,30 @@ The query parameters are not expected.
 
 | Name | Type | Description | Required | Example |
 | ---- | ---- | ------------| :------: | ------- |
-| accountStatus | `enum` | The new user’s [Account Status](./users_schemas.md/#account-status). | &#9745; | "accountStatus": "Suspended" |
+| firstName | `string` | The user’s new first name. | &#9744; | "firstName": "Jane" |
+| lastName | `string` | The user’s new last name. | &#9744; | "lastName": "Jordan" |
+| email | `string` | The user’s new email. | &#9744; | "email": "tom.jordan<span>@sample.</span>com" |
+| accountStatus | `enum` | The user’s new [Account Status](./users_schemas.md/#account-status). | &#9744; | "accountStatus": "Suspended" |
+| city | `string` | The user’s new city. | &#9744; | "city": "Brisbane" |
+| country | `string` | The user’s new country. | &#9744; | "country": "New Zeland" |
+| dateOfBirth | `string` | The user’s new dateOfBirth. The format needs to be in dd/MM/yyyy. | &#9744; | "dateOfBirth": "19/09/1999" |
+| department | `string` | The user’s new department. | &#9744; | "department": "Marketing" |
+| phoneNumberLand | `string` | The user’s new phoneNumberLand. | &#9744; | "phoneNumberLand": "Brisbane" |
+| phoneNumberMobile | `string` | The user’s new phoneNumberMobile. | &#9744; | "phoneNumberMobile": "Brisbane" |
+| timezone | `string` | The user’s new [timezone](./users_schemas.md/#time-zone). | &#9744; | "timezone": "Brisbane" |
```

Comment:
@WMKhant examples for phoneNumberLand, phoneNumberMobile, timezone do not look right.

Reply:
fixed it.